projects/demo/src/modules/app/customization/customization.component.ts

```diff
@@ -81,7 +81,7 @@ export class TuiCustomizationComponent implements AfterViewInit {
     onModelChange(variable: string, value: string | number) {
         this.variables = {
             ...this.variables,
-            [variable]: typeof value === 'string' ? value : value + 'px',
+            [variable]: typeof value === 'string' ? value : `${value}px`,
```

Comment:
We have a utility function `px` for that.

Reply:
fixed